CoreFunctionality/src/au/gov/asd/tac/constellation/functionality/startup/ConstellationLAFSettings.java

```diff
@@ -0,0 +1,333 @@
+        } else {
+            activeBlue = new Color(150, 200, 255);
+            activeDarkBlue = new Color(90, 125, 180);
+            selectedBlue = new Color(140, 180, 245);
+            selectedDarkBlue = new Color(75, 115, 170);
+            unselectedGreyBlue = new Color(190, 205, 215);
+            unselectedDarkGreyBlue = new Color(150, 160, 175);
+        }
+        
+        UIManager.getLookAndFeelDefaults().put("TabbedPane:TabbedPaneTab[MouseOver+Selected].backgroundPainter", 
+                new NimbusCustomGradientTabPainter(activeBlue, activeDarkBlue));
+        UIManager.getLookAndFeelDefaults().put("TabbedPane:TabbedPaneTab[Selected].backgroundPainter", 
+                new NimbusCustomGradientTabPainter(selectedBlue, selectedDarkBlue));
+        UIManager.getLookAndFeelDefaults().put("TabbedPane:TabbedPaneTab[Enabled].backgroundPainter", 
+                new NimbusCustomGradientTabPainter(unselectedGreyBlue, unselectedDarkGreyBlue));
+        
+        UIManager.getLookAndFeel().uninitialize();
+        UIManager.getLookAndFeel().initialize();
+    }
+
+    /**
+     * Applies a Blue color theme for the main window tabs when using the
+     * <b>FlatLaf</b> Look and Feel.
+     *
+     * @param darkMode When true, allocates Look and Feel tab colors suitable for 
+     * <u>FlatLafDark</u>, otherwise allocates colors suitable for <u>FlatLafLight</u>
+     */
+    private static void initFlatLafTabColors(boolean darkMode) {
+        Color selectedUnderlineBlue,
+                inactiveUnderlineBlue,
+                selectedBackgroundBlue,
+                hoverBackgroundBlue;
+
+        if (darkMode) {
+            selectedUnderlineBlue = new Color(30, 110, 190);
+            inactiveUnderlineBlue = new Color(20, 100, 175);
+            selectedBackgroundBlue = new Color(30, 60, 95);
+            hoverBackgroundBlue = new Color(45, 75, 115);
+        } else {
+            selectedUnderlineBlue = new Color(105, 145, 240);
+            inactiveUnderlineBlue = new Color(95, 130, 185);
+            selectedBackgroundBlue = new Color(200, 220, 255);
+            hoverBackgroundBlue = new Color(190, 210, 255);
+        }
+
+        UIManager.put("ViewTab.selectedBackground", selectedBackgroundBlue);
+        UIManager.put("ViewTab.hoverBackground", hoverBackgroundBlue);
+        UIManager.put("ViewTab.underlineHeight", 5);
+        UIManager.put("ViewTab.underlineColor", selectedUnderlineBlue);
+        UIManager.put("ViewTab.inactiveUnderlineColor", inactiveUnderlineBlue);
+
+        UIManager.put("EditorTab.selectedBackground", selectedBackgroundBlue);
+        UIManager.put("EditorTab.hoverBackground", hoverBackgroundBlue);
+        UIManager.put("EditorTab.underlineHeight", 5);
+        UIManager.put("EditorTab.underlineColor", selectedUnderlineBlue);
+        UIManager.put("EditorTab.inactiveUnderlineColor", inactiveUnderlineBlue);
+
+        try {
+            Class<?> tabDisplayer = (Class<?>) UIManager.getDefaults()
```

Comment:
I won't go through the rest of the finals but if you're able to add them to exceptions, variables, and parameters where possible, that would be great